### ISSUE TITLE
Search function for blueprint to include all project folder by default

### ISSUE THREAD

**Issue by IronSuit** [alexhajdu/fix_this#11]
When i'm trying to implement new systems i always put them into sub-folders to keep stuff organized, and it's annoying to include every sub-folder to let the plugin find the comment.

It would be great if the plugin would automatically search all of the folders for blueprints comments to add them in TODO.

Also the options to exclude certain folders would be very good, because i believe more people would like the ability to exclude certain folders, rather than the current setup, which is only include certain folders and leave all the rest.

**Comment by alexhajdu** [comment]
Yes, this is requested for some time as well. The solution for this will be to select just parent folders and plugin will search in those recursively.

**Comment by alexhajdu** [comment]
This will be released also in the next release.

**Comment by IronSuit** [comment]
Awesome, Cannot wait.

**Comment by IronSuit** [comment]
Has this been added yet?

**Comment by alexhajdu** [comment]
> Has this been added yet?

It's ready, but it was not shipped yet.
